prefetch.bs

```diff
@@ -141,6 +141,9 @@ The user agent may remove elements from the [=prefetch buffer=] even if they are
         1. Otherwise, perform [=HTTP-redirect fetch=] using |request| and |response|.
         1. Wait for the [=task=] on the [=networking task source=] to [=process response=] and set |response| to the result.
         1. If the result of |shouldBlockNavigationResponse| given |request| and |response| is "`Blocked`", then set |response| to a [=network error=] and [=iteration/break=].
+        1. If |response| is not a [=network error=], |browsingContext| is a [=child browsing context=], and the result of performing a [=cross-origin resource policy check=] with |browsingContext|'s [=browsing context/container document=]'s [=Document/origin=], |browsingContext|'s [=browsing context/container document=]'s [=relevant settings object=], |request|'s [=request/destination=], |response| and true is <strong>blocked</strong>, then set |response| to a [=network error=] and [=iteration/break=].
+
+            <div class="note">Here we're running the [=cross-origin resource policy check=] against the [=parent browsing context=] rather than sourceBrowsingContext. This is because we care about the same-originness of the embedded content against the parent context, not the navigation source.</div>
```

Comment:
```suggestion
            <div class="note">Here we're running the [=cross-origin resource policy check=] against the [=parent browsing context=] rather than |sourceBrowsingContext|. This is because we care about the same-originness of the embedded content against the parent context, not the navigation source.</div>
```